api/serverpb/server.proto

```diff
@@ -111,11 +111,18 @@ message MetricsResolutions {
   google.protobuf.Duration lr = 3;
 }
 
+// QAN contains query analytics configuration.
+message QAN {
+  // Number of days of data retaintion in format: '24h'.
```

Comment:
FYI, https://developers.google.com/protocol-buffers/docs/proto3#json:
> Generated output always contains 0, 3, 6, or 9 fractional digits, depending on required precision, **followed by the suffix "s"**. Accepted are any fractional digits (also none) as long as they fit into nano-seconds precision and **the suffix "s" is required**.

We can't change that on the API level, but we will make it nicer in CLI and UI.

I will update that comment.